packages/components/src/components/Table/Table.stories.tsx

```diff
@@ -160,7 +181,8 @@ export const ReactTable = (): JSX.Element => {
     <Table>
       <THead>
         {map(table.getHeaderGroups(), (headerGroup) => (
-          <Tr key={headerGroup.id}>
+          // It seems like a <tr> tag is essential for react table. But notice I do not have the <tr> tag in the above examples. Do we need it?
+          <tr key={headerGroup.id}>
```

Comment:
Tables will work without a `tr`. Browsers can figure it out, but our table should have a `tr`. Its the proper way of doing things.